examples/fsm_walking_compiled_controller.py

```diff
@@ -170,35 +198,35 @@
                 outputs.time_in_current_state,
                 outputs.knee_impedance.eq_angle,
                 outputs.ankle_impedance.eq_angle,
-                osl.loadcell.fz,
+                loadcell.fz,
             ),
             end="\r",
         )
 
         # Write to the hardware
-        osl.knee.set_joint_impedance(
-            K=units.convert_to_default(
+        knee.set_joint_impedance(
```

Comment:
I think this release would be a good time to refactor all of the "joint" references with "output" or something like that. We have "output" position, but "joint impedance"

Reply:
Agreed, renaming all set_joint_impedance methods to set_output_impedance. Do you think a deprecated warning with routing is necessary here?